ColorStripe/Views/Device/DeviceView.swift

```diff
@@ -13,7 +13,7 @@ struct DeviceView: View {
 
     //MARK: - Lifecycle
     
-    init(peripheral: CBPeripheral) {
+    init(peripheral: PeripheralModel) {
```

Comment:
We can remove extra `import CoreBluetooth` now in view models and views. Please check all cases.

Reply:
Done